usermanager/src/main/java/org/pesmypetcare/usermanager/datacontainers/user/UserDataSender.java

```diff
@@ -0,0 +1,85 @@
+package org.pesmypetcare.usermanager.datacontainers.user;
+
+import androidx.annotation.NonNull;
+
+import java.util.Objects;
+
+/**
+ * @author Santiago Del Rey
+ */
+public class UserDataSender {
+    private String uid;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'uid' could be replaced by a local variable.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5650085)